content/posts/google_sheets_as_apis.md

```diff
@@ -0,0 +1,148 @@
+---
+title: Accessing Google Sheets Data from Genezio Using a Service Account
```

Comment:
for SEO reasons, I suggest to use "how to" in the title: "How to Access Google Sheets Data Using a Service Account"